### ISSUE TITLE
Email Validation

### ISSUE THREAD

**Issue by lonejack** [rainlab/user-plugin#527]
Hi, I've installed this plugin. It works as expected but after login. If I enter in the account page, I'm allowed to change the email without re-validation.
Just to be clear. My account page appear in this way:
Registration/Login appears in this way:
![image](https://github.com/rainlab/user-plugin/assets/933876/77257b5d-eaba-446b-98b6-7e52c138d5d6)
After Registration it is requested to validate the email: OK!
Login works fine: OK!

But, after login, when I return in the account page I see this:
![image](https://github.com/rainlab/user-plugin/assets/933876/bafe3b9f-97a1-428d-a2ce-3c287920449c)

I can change the email without a new email validation!
Any workaround?

**Comment by daftspunk** [comment]
This is a known issue with the plugin, we will fix this in v2 so leave this ticket open.

As a workaround, you could hook the user model like this

```php
User::extend(function($model) { 
    $model->bindEvent('model.beforeSave', function() use ($model) { 
        if ($model->isDirty('email')) {
            $model->activated_at = null;
        }
    });
});
```

**Comment by daftspunk** [comment]
This is fixed in v3 (#526) and will be released soon.

When the user updates their email, the action is logged, and they must verify it again.
